common-types/src/main/scala/com/improving/app/common/errors/Validation.scala

```diff
@@ -6,7 +6,7 @@ object Validation {
 
   type Validator[T] = T => Option[ValidationError]
 
-  def applyAllValidators[T](validators: Seq[Validator[T]]): Validator[T] =
+  def applyAllValidators[T](validators: Validator[T]*): Validator[T] =
```

Comment:
Never seen this `*` notation before - I assume it's related to `: _*` notation when passing a list as parameters?

Reply:
Yes, I think it's the same.